style.css

```diff
@@ -613,3 +621,229 @@ footer .btn:active {
   color: white;
   background: #2230d2;
 }
+
+@media screen and (min-width: 768px) {
+  #toolbar {
+    display: flex;
+    justify-content: space-around;
+    align-items: space-around;
+  }
+
+  .burger-menu {
+    display: none;
+    height: 0;
+    width: 0;
+    margin: 0;
+    padding: 0;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  nav ul {
+    width: 60%;
+    display: flex;
+    justify-content: flex-end;
+    gap: 24px;
+  }
+
+  nav ul li {
+    text-decoration: none;
+    list-style-type: none;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    color: #344563;
+  }
+
+  .logo-nav {
+    margin: 0 0 0 120px;
+  }
+
+  .hero {
+    align-items: center;
+    justify-content: center;
+    background-image: url(./desktopPics/Hero\ Desktop\ Background.svg);
+    background-size: 100% auto;
+    height: 916px;
+  }
+
+  .hero h1 {
+    width: 70%;
+    font-size: 48px;
+    color: #091e42;
+    font-weight: 700;
+  }
+
+  footer p {
+    width: 569px;
+    height: 102px;
+  }
+
+  .hero p {
+    width: 70%;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    color: #344563;
+  }
+
+  .hero h6 {
+    margin-right: 380px;
+  }
```

Comment:
- [ ] Please try to fix this issue below since it effects user experience that when the screen width goes larger, this part goes to the right: 
![image](https://user-images.githubusercontent.com/91486791/194409589-3418e3b9-f521-47b8-8088-f2acf550cf6f.png)

Reply:
Thank you for identifying this crucial error. I have corrected it🤗.